packages/api/src/sequelize/migrations/2023-12-12_00_create-patient-discovery.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from "sequelize";
+import type { Migration } from "..";
+import * as shared from "../migrations-shared";
+
+const tableName = "patient_discovery_result";
+
+// Use 'Promise.all' when changes are independent of each other
+export const up: Migration = async ({ context: queryInterface }) => {
+  await queryInterface.sequelize.transaction(async transaction => {
+    await shared.createTable(
+      queryInterface,
+      tableName,
+      {
+        id: {
+          type: DataTypes.STRING,
+          primaryKey: true,
+        },
```

Comment:
Do we need ID on this table? Asking b/c it generates an index and I'm not sure we need an ID. Or could create the ID as UUID and not mark it as PK (so no index).

Reply:
I believe we still want the id but ill remove the primarykey